releases/client/3.10/settings

```diff
@@ -1 +1 @@
-OSES="el7 el8 el9"
+OSES='el7 el8 el9'
```

Comment:
Pedantic, but why the quote change?

Reply:
because locally i had ''' instead of '"', noticed just now as well, fixing, as technically it should be unmodified with the removal of the foreman version

Reply:
Should be fixed now to only change 1 file.